kiwi/system/identifier.py

```diff
@@ -32,6 +36,10 @@ class SystemIdentifier:
     """
     def __init__(self):
         self.image_id = None
+        sde = os.environ.get('SOURCE_DATE_EPOCH')
+        if sde:
+            log.info('Using SOURCE_DATE_EPOCH as random seed.')
```

Comment:
can we change this to

log.info(f'Using SOURCE_DATE_EPOCH as random seed: {sde}')

Reply:
Yes please, we need to know what value is used.

Reply:
Done